### DISCUSSION TITLE
DataTable Filter Variants in PrimeOne-3.0.1

### DISCUSSION THREAD

**Discussion by diazemiliano** [primefaces/community#3060] [Figma UI Kit] [answered]
Hi I Just bought the license and I'm using the `PrimeOne-3.0.1` version, in which I can't find the way to enable the `DataTable` filters (https://primevue.org/datatable/#filter) in the Figma Component. Is there any way?
Thanks in Advance.

**Comment by onursenture** [comment, marked as answer]
Hello, this feature is missing from the Table component in PrimeOne. We're collecting feedback for future versions, and this could be one of the features we add. Thank you for your feedback.
